src/rust-crypto/backup.ts

```diff
@@ -30,14 +32,26 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
     /** Have we checked if there is a backup on the server which we can use */
     private checkedForBackup = false;
     private activeBackupVersion: string | null = null;
+    private stopped = false;
+
+    /** whether {@link backupKeysLoop} is currently running */
+    private backupKeysLoopRunning = false;
 
     public constructor(
         private readonly olmMachine: OlmMachine,
         private readonly http: MatrixHttpApi<IHttpOpts & { onlyData: true }>,
+        private readonly outgoingRequestProcessor: OutgoingRequestProcessor,
     ) {
         super();
     }
 
+    /**
+     * Tell the RustBackupManager to immediately stop.
```

Comment:
This comment doesn't quite match reality, it stops it on the next loop and may still process & emit events after this is called.